RFC-676-Contributor-Council-Charter.md

```diff
@@ -0,0 +1,221 @@
+# CDK Contributor Council Charter (RFC) — November, 2024
+
+## Purpose of the Council
+
+To improve visibility into CDK, increase feedback opportunities, and build a stronger, more inclusive community, the AWS CDK team proposes forming a
+Contributor Council.
+
+*Original Author(s)*: [@haubles](https://github.com/haubles), [@billfine](https://github.com/billfine)
+
+*Tracking Issue*: https://github.com/aws/aws-cdk-rfcs/issues/676
+
+## Principles guiding this RFC
+
+1. **Maintain AWS supportability.** In order for AWS to provide on-call resources tasked with resolving high-impact bugs or issues in CDK for
+customers and the community, AWS must ensure that its standards for processes—for example, security reviews—are met. Therefore, AWS controls
+the CDK release process.
```

Comment:
How will AWS support the code that is being written by the community that will be part of the library?   It seems it needs a lot more people and also it will need to write the code in the first place.  If it does'tn have resource to write the code, how can you support the code.

Reply:
That is already the case, and I do not understand how that changes due to the council.

Reply:
If the result is more code, in the library, there is more support required. If you cant' support it, AWS won't be able to deliver its contracted responsiblities to those who pay for it. ( Enterprise Support ).